src/services/services.ts

```diff
@@ -1665,6 +1665,11 @@ namespace ts {
             return true;
         }
 
+        function isValidSnippetInsertionAtPosition(fileName: string, position: number): boolean {
+            const completionInfo = getCompletionsAtPosition(fileName, position);

```

Comment:
After discussion with @vladima & @mhegazy it makes sense to merge our regular & snippet completion providers in VS as it is not worth the effort to extract the syntactic information to determine this on the script side.  Closing this review out and will follow up with a separate review on the VS host side.